tools/debug-ui/src/Tier1View.tsx

```diff
@@ -0,0 +1,188 @@
+import { MouseEvent, ReactElement, useCallback, useMemo, useState } from "react";
+import { useQuery } from "react-query";
+import { fetchEpochInfo, fetchFullStatus, PeerInfoView } from "./api";
+import { formatDurationInMillis, formatTraffic, addDebugPortLink } from "./utils";
+import "./Tier1View.scss";
+
+type Tier1ViewProps = {
+    addr: string,
+};
+
+type PeerInfo = {
+    peer: PeerInfoView,
+    validator: string[],
+    routedValidator: string[],
+    statusClassName: string,
+}
+
+export const Tier1View = ({ addr }: Tier1ViewProps) => {
+    const { data: fullStatus, error: fullStatusError, isLoading: fullStatusLoading } =
+        useQuery(['fullStatus', addr], () => fetchFullStatus(addr));
+
+    if (fullStatusLoading) {
+        return <div>Loading...</div>;
+    }
+    if (fullStatusError) {
+        return <div className="network-info-view">
+            <div className="error">
+                {(fullStatusError as Error).stack}
+            </div>
+        </div>;
+    }
+    if (!fullStatus) {
+        return <div className="network-info-view">
+            <div className="error">No Data</div>
+        </div>;
+    }
+
+    let rendered = new Set();
+    const network_info = fullStatus.detailed_debug_status!.network_info;
+
+    let rows = new Array();
+
+    // tier1_connections contains TIER1 nodes we are currently connected to
+    network_info.tier1_connections!.forEach(function (peer) {
+        let account_key = "";
+        let proxies = new Array();
+
+        network_info.tier1_accounts_data!.forEach(account => {
```

Comment:
nit: There isn't a compelling reason to use forEach unless you also need the index. You can simply do `for (const account of network_info.tier1_accounts_data!)`